tests/unit/test_group_project.py

```diff
@@ -35,6 +36,10 @@ def _make_workgroup(user_ids):
     return WorkgroupDetails(users=[{'id': user_id} for user_id in user_ids])
 
 
+def _convert_string_to_time(datetime_string):
```

Comment:
@ziafazal nit: please move to test.utils. Also, a better name might be `parse_date`.